static/bidder-info/oraki.yaml

```diff
@@ -0,0 +1,18 @@
+endpoint: "https://eu1.oraki.io/pserver"
+maintainer:
+  email: "ben@oraki.io"
```

Comment:
`ben@oraki.io` seems associated to an individual. Prebid recommends to use an group or organization email as maintainer email.